job_definitions/stats_snapshots.yml

```diff
@@ -12,13 +12,6 @@
     logrotate:
       daysToKeep: 20
       artifactDaysToKeep: 20
-    scm:
-      - git:
-          url: git@github.com:alphagov/digitalmarketplace-api.git
```

Comment:
Does this not need to be replaced by scripts repo?

Reply:
Nah. Docker is installed straight onto the Jenkins box. It pulls and runs the latest image of dm-scripts from Dockerhub, which contains the repo.